packages/api/src/elastic/pages.ts

```diff
@@ -148,6 +148,50 @@ const appendInFilter = (builder: ESBuilder, filter: InFilter): ESBuilder => {
             gte: 'now-14d',
           },
         })
+    case InFilter.LIBRARY:
+      return builder.query('nested', {
+        path: 'labels',
+        query: {
+          bool: {
+            should: [
```

Comment:
Does this mean it would need the `library` label to return true?

Reply:
@jacksonh not really. it will be true if it has `library` label or it does not have either `newsletter` or `rss` label

Reply:
great, thanks! Looks good!